docs/advanced_usage.rst

```diff
@@ -41,6 +41,28 @@ We can substitute this function directly for the heat_loss_to_room model compone
 
 If you want to replace a subscripted variable, you need to ensure that the output from the new function is the same as the previous one. You can check the current coordinates and dimensions of a component by using :py:data:`.get_coords(variable_name)` as it is explained in :doc:`basic usage <../basic_usage>`.
 
-Supplying additional arguments to the integrator
-------------------------------------------------
-the :py:func:`run` function's argument `intg_kwargs` is a pass-through for keyword arguments to scipy's odeint function, and as such can take on any of the keywords that odeint recognizes.
+Starting simulations from an end-state of another simulation
+------------------------------------------------------------
+The current state of a model can be saved in a pickle file using the :py:data:`.export()`method::
+
+   model.run(final_time=50)
+   model.export("final_state.pic")
+
+Then the exported data can be used in another session::
+
+   model.run(initial_condition="final_state.pic", return_timestamps=[55, 60])
+
+the new simulation will have initial time equal to 50 with the saved values from the previous one.
+
+.. note::
+   You can set the exact final time of the simulation using the *final_time* argument.
+   If you want to avoid returning the dataframe of the stocks you can use *return_timestamps=[]*::
+
+     model.run(final_time=50, return_timestamps=[])
+
+.. note::
+   The changes done with *params* arguments are not saved. If you want to keep them, you need to call run with the same *params* values as in the original simulation.
```

Comment:
Changes made with params are not ported to the new model object that you initialize with `final_state.pic` - but the original model object still has the parameters that you assigned using param. Maybe would be clearer to use something like: 
```python
model1 = pymc.read_vensim(...)
model1.run(...)
model1.export("final_state.pic")

model2 = pymc.read_vensim(...)
model2.run(initial_condition="final_state.pic", ...)
```